nipype/interfaces/cat12/preprocess.py

```diff
@@ -0,0 +1,363 @@
+    approach.
+    The segmentation approach is based on an Adaptive Maximum A Posterior (MAP) technique without the need for a priori
+    information about tissue probabilities. That is, the Tissue Probability Maps (TPM) are not used constantly in the
+    sense of the classical Unified Segmentation approach (Ashburner et. al. 2005), but just for spatial normalization.
+    The following AMAP estimation is adaptive in the sense that local variations of the parameters (i.e., means and
+    variance) are modeled as slowly varying spatial functions (Rajapakse et al. 1997). This not only accounts for
+    intensity inhomogeneities but also for other local variations of intensity.
+    Additionally, the segmentation approach uses a Partial Volume Estimation (PVE) with a simplified mixed model of at
+    most two tissue types (Tohka et al. 2004). We start with an initial segmentation into three pure classes: gray
+    matter (GM), white matter (WM), and cerebrospinal fluid (CSF) based on the above described AMAP estimation. The
+    initial segmentation is followed by a PVE of two additional mixed classes: GM-WM and GM-CSF. This results in an
+    estimation of the amount (or fraction) of each pure tissue type present in every voxel (as single voxels - given by
+    Another important extension to the SPM12 segmentation is the integration of the Dartel or Geodesic Shooting
+    registration into the toolbox by an already existing Dartel/Shooting template in MNI space. This template was
+    derived from 555 healthy control subjects of the IXI-database (http://www.brain-development.org) and provides the
+    several Dartel or Shooting iterations. Thus, for the majority of studies the creation of sample-specific templates
+    is not necessary anymore and is mainly recommended for children data.'};
+
+    http://www.neuro.uni-jena.de/cat12/CAT12-Manual.pdf#page=15
+
+    Examples
+    --------
+    >>> path_mr = 'structural.nii'
+    >>> cat = CAT12Segment(in_files=path_mr)
+    >>> cat.run() # doctest: +SKIP
+    """
+
+    in_files = InputMultiPath(ImageFileSPM(exists=True), field="data", desc="file to segment", mandatory=True,
+                              copyfile=False)
+
+    help_tpm = 'Tissue Probability Map. Select the tissue probability image that includes 6 tissue probability ' \
+               'classes for (1) grey matter, (2) white matter, (3) cerebrospinal fluid, (4) bone, (5) non-brain soft ' \
+               'tissue, and (6) the background.  CAT uses the TPM only for the initial SPM segmentation.'
+    tpm = InputMultiPath(ImageFileSPM(exists=True), field="tpm", desc=help_tpm, mandatory=False,
+                         copyfile=False)
+
+    n_jobs = traits.trait_types.Int(1, usedefault=True, mandatory=True, field="nproc", desc="Number of threads")
+    use_prior = traits.trait_types.Str(field="useprior", usedefault=True)
```

Comment:
Based on changed imports:

```suggestion
    n_jobs = traits.Int(1, usedefault=True, mandatory=True, field="nproc", desc="Number of threads")
    use_prior = Str(field="useprior", usedefault=True)
```

And can you add a description to `use_prior`?